packages/core/bootstrap/src/lib/ws/metrics.ts

```diff
@@ -0,0 +1,43 @@
+import * as client from 'prom-client'
+
+export const ws_connection_active = new client.Gauge({
```

Comment:
We can probably remove all the experimental labels since all metrics are experimental currently